src/main/java/io/aiven/connect/jdbc/sink/BufferedRecords.java

```diff
@@ -153,14 +169,14 @@ private void reInitialize(final SchemaPair schemaPair) throws SQLException {
     }
 
     public List<SinkRecord> flush() throws SQLException {
-        if (records.isEmpty()) {
-            log.debug("Records is empty");
+        if (records.isEmpty() && tombstoneRecords.isEmpty()) {
+            log.debug("Records are empty.");
```

Comment:
nit
I am not sure about this debug.
- the debug text should include Records and tombstone records are empty. 
- Do we  want to have a debug is the records are empty as before and there are tombstones .

Reply:
> Do we want to have a debug is the records are empty as before and there are tombstones.

It seems redundant to include if statements solely for debugging empty records, as the prepare statement debug (mentioned previously) will already cover this aspect.